### PR TITLE
Updated Getting Started to include vtadmin link.

### PR DESCRIPTION
**Pull request by andylim-duo**
### Description
This PR updates the Getting Started documentation for the [Local Install Via Docker](http://localhost:1313/docs/16.0/get-started/local-docker/) page. This update is related to [PR #12467](https://github.com/vitessio/vitess/pull/12467) on the vitess project.

### Testing
The update was manually verified and shown in the screen capture below:

The-Vitess-Docs-Local-Install-via-Docker

### CODE REVIEW

**Review by mattlord — COMMENTED** [review]
LGTM! Do you mind making the same change on the v14, v15, and v17 pages too?

**Comment by andylim-duo** [comment]
> LGTM! Do you mind making the same change on the v14, v15, and v17 pages too?

Sure thing. I'll update those versions.

File: content/en/docs/14.0/get-started/local-docker.md [hunk 1]
```diff
@@ -37,11 +37,13 @@ In your shell, execute:
 make docker_run_local
 ```
 
-This will set up a MySQL replication topology, as well as `etcd`, `vtctld` and `vtgate` services. 
+This will set up a MySQL replication topology, as well as `etcd`, `vtctld`, `vtgate`,
```
Comment (mattlord) [inline content/en/docs/14.0/get-started/local-docker.md:40]: Nit, but missing the `and` here.

Comment (andylim-duo) [reply content/en/docs/14.0/get-started/local-docker.md:40]: Yup, you beat me to the update.

**Review by mattlord — APPROVED** [review]
❤️ @notfelineit, can you please take a look too?

**Review by notfelineit — APPROVED** [review]
🎉 great work

**Pull request merged**